File: LatPrak3_spekPc (JAVA)/Disk.java
class Disk{
	//atribut
	private String type;
	private String capacity;
	private int price;

	//constructor
	Disk(){
	}

	//get & set untuk atribut type
	public void setType(String type){
		this.type = type;
	}
		
	public String getType(){
		return this.type;
	}

	//get & set untuk atribut capacity
	public void setCapacity(String capacity){
		this.capacity = capacity;
	}
		
	public String getCapacity(){
		return this.capacity;
	}

	//get & set untuk atribut price 
	public void setPrice(int price){
		this.price = price;
	}
		
	public int getPrice(){
		return this.price;
	}
}

File: LatPrak3_spekPc/Disk.cpp
#include <string>

using namespace std;

class Disk{
	private:
	//atribut dalam class disk
		string type;
		string capacity;
		int price;

	public:
		//constructor
		Disk(){
		}

		Disk(string type, string capacity, int price){
			this->type = type;
			this->capacity = capacity;
			this->price = price;
		}


		//prosedur dan fungsi untuk get & set dari atribut type
		void setType(string type){
			this->type = type;
		}
		
		string getType(){
			return this->type;
		}

		//prosedur dan fungsi untuk get & set dari atribut capacity
		void setCapacity(string capacity){
			this->capacity = capacity;
		}
		
		string getCapacity(){
			return this->capacity;
		}

		//prosedur dan fungsi untuk get & set dari atribut price
		void setPrice(int price){
			this->price = price;
		}
		
		int getPrice(){
			return this->price;
		}

	//destructor
	~Disk(){
	}
};